cachematrix.R

```diff
@@ -2,14 +2,35 @@
 ## functions do
 
 ## Write a short comment describing this function
-
-makeCacheMatrix <- function(x = matrix()) {
-
+### First step is to create a function "makeCacheMatrix ### this function will keep inverse of the matrix
+makeCacheMatrix <- function(mtx = matrix()) {
+  invs <- NULL    
+  ## define set function when a new matrix is created  
+       set <- function(mty) {    
+               mtx <<- mty        ## assigning value to parent variable     
+               invs <<- NULL      ## to reset parent variable to NULL to get inserve calulated  
+       }    
+  ## define get function   
+        get <- function() mtx    
+  ## setinverse function will prepare inverse of the matrix   
+        setinverse <- function(inverse) invs <<- inverse    
+  ## getinverse function will return inverse from cache    
+        getinverse <- function() invs    
+  ## return list to calling function   
+        list(set = set,       get = get,       setinverse = setinverse,       getinverse = getinverse)
 }
 
-
 ## Write a short comment describing this function
-
-cacheSolve <- function(x, ...) {
```

Comment:
again same problem.. in editing, this line extra came here..  how to remove it..